o2a/converter/oozie_converter.py

```diff
@@ -111,14 +111,17 @@ def convert_nodes(self):
         """
         logging.info("Converting nodes to tasks and inner relations")
         for p_node in self.workflow.nodes.values():
+            if p_node.tasks:
+                continue
             tasks, relations = p_node.mapper.to_tasks_and_relations()
             p_node.tasks = tasks
             p_node.relations = relations
 
     def convert_dependencies(self) -> None:
         logging.info("Converting dependencies.")
         for node in self.workflow.nodes.values():
-            self.workflow.dependencies.update(node.mapper.required_imports())
+            if node.mapper:
+                self.workflow.dependencies.update(node.mapper.required_imports())
```

Comment:
I'd suggest implementing a default `required_imports()` in the `BaseMapper` returning an empty set.

Reply:
This code should not change the behavior of the mapper. The change stems from the attempt to loosen relation between Mapper in ParsedActionNoe.